README.md

```diff
@@ -6,6 +6,7 @@
 
 - Use at least 1 programming code. You can use any kind of programming languages.
 - Use at least 1 database which can use SQL or SQL-like query, such as PostgreSQL, MySQL, sqlite3, Google BigQuery, InfluxDB, etc...
+- Consider the real-world problems you think (e.g. performance, maintainability)
```

Comment:
Take into consideration problems you may face in the real-world (e.g. performance, maintainability etc)